site/patches/ajv+8.9.0.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/node_modules/ajv/dist/core.js b/node_modules/ajv/dist/core.js
+index dbc417b..2f3198d 100644
+--- a/node_modules/ajv/dist/core.js
++++ b/node_modules/ajv/dist/core.js
+@@ -549,8 +549,8 @@ function checkKeyword(keyword, def) {
+     (0, util_1.eachItem)(keyword, (kwd) => {
+         if (RULES.keywords[kwd])
+             throw new Error(`Keyword ${kwd} is already defined`);
+-        if (!KEYWORD_NAME.test(kwd))
+-            throw new Error(`Keyword ${kwd} has invalid name`);
++        // if (!KEYWORD_NAME.test(kwd))
++        //     throw new Error(`Keyword ${kwd} has invalid name`);
```

Comment:
It’d be helpful to introduce a comment explaining the change

Reply:
Done